api/v1alpha1/httproute_types.go

```diff
@@ -230,6 +230,21 @@ type HTTPRouteAction struct {
 	// that includes the HTTPRoute will be true.
 	ForwardTo *ForwardToTarget `json:"forwardTo" protobuf:"bytes,1,opt,name=forwardTo"`
 
+	// MirrorTo sends a copy of the request to the referenced object. A
+	// response is not expected from the MirrorTo target. The resource
+	// may be "services" (omit or use the empty string for the group),
+	// or an implementation may support other resources (for example,
+	// resource "myroutetargets" in group "networking.acme.io"). Omitting
+	// or specifying the empty string for both the resource and group
+	// indicates that the resource is "services".  If the referent cannot
+	// be found, the "InvalidRoutes" status condition on any Gateway that
+	// includes the HTTPRoute will be true.
+	//
+	// Support: core
```

Comment:
Can we elaborate that support for services is in the core and others are custom?

I think we have done a good job for `TargetRef` that we can probably reuse:
```
	// Support: Core (Kubernetes Services)
	// Support: Implementation-specific (Other resource types)
```

Reply:
ack